src/RosClientNode.hpp

```diff
@@ -162,6 +179,8 @@ class RosClientNode : public QObject {
             publish_ros_msg<T>(msg, msg_type, full_to_topic);
           };
     }
+    
+    pub_remote_topics.push_back(full_from_topic);
```

Comment:
does this do what you want? Doesn't this get you subscriptions to the un-namespaced (assuming they are absolute names) local topics, which you probably won't actually get from the server? Without knowing what namespaces the other robots have, you could maybe do something like prepend it with a `.*` to capture all namespaces, but I'm not sure what the intended behavior is.

Not opposing the rest of the code; at worst you could have another function called at config time that accepts a regex and adds it to this vector.

Reply:
full_from_topic is the fully resolved result of whatever the user specifies as the topic to expect the server is sending them, since this is in the `register_remote_topic` function. In the example config, this is non-absolute for the default topics, which should have the correct behaviour where the namespace gets prepended to what is in the config. Therefore, full_from_topic should be what the server sends you

Reply:
I see, GitHub failed me here and I thought it was register_msg_type since it looks similar haha. Looks good to me!